skyportal/plot.py

```diff
@@ -342,7 +342,7 @@ def photometry_plot(obj_id, user, width=600, device="browser"):
 
     labels = []
     for i, datarow in data.iterrows():
-        label = f'{datarow["instrument"]}/{datarow["filter"]}'
+        label = f'{datarow["instrument"]}/{datarow["filter"]}asfasdfasfsafdsa'
```

Comment:
Is this supposed to be here?

Reply:
Oops, no that's leftover test code good catch